### DISCUSSION TITLE
Como calificar una tarea grupal?

### DISCUSSION THREAD

**Discussion by emayorgacueva** [chamilo/chamilo-lms#5204] [General]
Buenas noches, un caso tengo un grupo de 5 estudiantes, el cual se envio una tarea al grupo, como hacer para que un integrante de ese grupo entregue la tarea y se califique al grupo en general, es posible hacer eso en Chamilo?

**Comment by ywarnier** [comment]
Hola @emayorgacueva 
Normalmente usas la herramienta de grupos *dentro* del curso. Ahí entras como profesor, defines la tarea para este grupo, y luego uno de los estudiantes del grupo puede subir la tarea, y la nota que le des se aplicará a todos los estudiantes.
Hace tiempo que no he probado esto, así que esta es la teoría pero hay que comprobarla.

**Comment by emayorgacueva** [comment]
Gracias ywarnier por contestar, pero he hecho lo que comentabas, pero al subir uno de los dos estudiantes que conforma el grupo, en que opcion se le aplica la nota a todos los estudiantes?

**Comment by contidos** [comment]
Hola! Yo lo he probado también y no funciona así. El espacio de tareas que existe en el grupo es idéntico al que existe en el curso y funciona de igual modo.  Es decir, como profesor, una vez has recibido la tarea por parte de un alumno, si quieres aplicarle esa misma nota al resto de alumnos, debes acceder al apartado de tarea de cada uno de ellos y corregirle la tarea. 

De hecho, no puedes tampoco corregirlo masivamente ya que si los estudiantes no han enviado la tarea, no saldrán en el listado. 

En resumen, todos tienen que enviar la tarea y tú debes abrir cada usuario y darle la puntuación.

**Comment by emayorgacueva** [comment]
Bueno esperemos que en esta nueva versión lo puedan incluir ya que si es de importancia esta funcionalidad, los docentes muchas veces califican de formal grupal a sus estudiantes.

**Reply by contidos** [reply]
Pues parece una buena idea como funcionalidad a añadir. Te animo a que hagas esta sugerencia como propuestas de mejora de la funcionalidad. Puedes hacerlo desde la sección de Issues. Allí detallas cómo funciona ahora y cómo te gustaría que funcionase la nueva versión y las mejoras que propones; y así el equipo de Chamilo puede considerarlo de cara a nuevas versiones.
